lib/src/rows/Rows.stories.tsx

```diff
@@ -0,0 +1,314 @@
+import React from "react";
+import Title from "../../.storybook/components/Title";
+import styled from "styled-components";
+import DxcRows from "./Rows";
+
+export default {
+  title: "Rows",
+  component: DxcRows,
+};
+
+export const Chromatic = () => (
+  <>
+    <Title title="Default" theme="light" level={4} />
```

Comment:
Is not the default value `strech`?

Reply:
Yes, but this example have a fixed width.

```
<DxcRows>
  <DxcRows.Row>
    <Placeholder width="small" />
  </DxcRows.Row>
  <DxcRows.Row>
    <Placeholder width="medium" />
  </DxcRows.Row>
  <DxcRows.Row>
    <Placeholder width="large" />
  </DxcRows.Row>
</DxcRows>
```